Cargo.toml

```diff
@@ -281,24 +281,24 @@ members = [
 default-members = ["cmd/tikv-server", "cmd/tikv-ctl"]
 
 [profile.dev.package.grpcio-sys]
-debug = false
-opt-level = 1
+debug = true
```

Comment:
This settings are added for a reason. They are used to make target directory small.

Reply:
My fault. I change it to debug and forget to change it back.